indexer/services/ender/src/scripts/handlers/dydx_stateful_order_handler.sql

```diff
@@ -22,9 +22,40 @@ DECLARE
     order_record orders%ROWTYPE;
     subaccount_record subaccounts%ROWTYPE;
 BEGIN
+    /* For order replacement, remove old order first and don't return immediately */
+    IF event_data->'orderReplacement' IS NOT NULL THEN
+        order_id = event_data->'orderReplacement'->'oldOrderId';
+        order_record."status" = 'CANCELED';
+
+        clob_pair_id = (order_id->'clobPairId')::bigint;
+        perpetual_market_record = dydx_get_perpetual_market_for_clob_pair(clob_pair_id);
+
+        subaccount_id = dydx_uuid_from_subaccount_id(order_id->'subaccountId');
+        SELECT * INTO subaccount_record FROM subaccounts WHERE "id" = subaccount_id;
+        IF NOT FOUND THEN
+            RAISE EXCEPTION 'Subaccount for order not found: %', order_;
+        END IF;
+
+        order_record."id" = dydx_uuid_from_order_id(order_id);
+        order_record."updatedAt" = block_time;
+        order_record."updatedAtHeight" = block_height;
+        UPDATE orders
+        SET
+            "status" = order_record."status",
+            "updatedAt" = order_record."updatedAt",
+            "updatedAtHeight" = order_record."updatedAtHeight"
+        WHERE "id" = order_record."id"
+        RETURNING * INTO order_record;
+
+        IF NOT FOUND THEN
+            RAISE NOTICE 'Unable to cancel replaced order because not found with orderId: %', dydx_uuid_from_order_id(order_id);
```

Comment:
what's the diff between notice and exception?

Reply:
raise notice doesn't abort the transaction. I basically just want to continue here if not found, do you know of a better way to do that?